spec/policies/course_policy_spec.rb

```diff
@@ -35,6 +39,10 @@
       it 'allows the :show action' do
         expect(subject.show?).to be_falsey
       end
+
+      it 'allows the :update action' do
```

Comment:
The words aren't quite right here.

Reply:
allows the :show action
Vs 
allows the :update action
Vs
... 

Not sure what's you mean...

Reply:
https://github.com/varvet/pundit/blob/master/spec/policies/post_policy_spec.rb


Actually the custom mishy matcher will get rid of my bad English

Reply:
Guess I need to look at this as well.

https://github.com/varvet/pundit/blob/master/spec/spec_helper.rb

Cheers misha